syntaxhighlighter3/styles/shCore.css

```diff
@@ -59,14 +59,18 @@
 }
 
 .syntaxhighlighter {
+  box-sizing: border-box;
   width: 100% !important;
   margin: 1em 0 1em 0 !important;
   position: relative !important;
-  overflow: auto !important;
+  overflow-x: scroll !important;
   overflow-y: hidden !important;
   font-size: 1em !important;
   padding: 0.5em 1em !important;
 }
+.entry-content:has(.syntaxhighlighter) {
```

Comment:
Ahh shoot, I was under the impression we'd standardized using `.entry-content` across themes. Since this issue is happening across multiple themes, I'll update the themes where we've found issues rather than trying to update all of them in one go. Thanks for the review!

Reply:
I would still propose we update the box-sizing here since it's going to break out of any fixed-width containers due to added padding. Thoughts?

Reply:
> I was under the impression we'd standardized using .entry-content across themes. 

That could definitely work. However, some non-A8C themes also use the `.entry-content` class as a standard class name. I wanted to avoid any potential side effects.